### ISSUE TITLE
Overload error in `bridge_generated.dart`

### ISSUE THREAD

**Issue by w-ensink** [fzyzcjy/flutter_rust_bridge#503]
I followed all steps in the [tutorial](http://cjycode.com/flutter_rust_bridge/integrate.html), specifically the steps for macOS. I keep getting a function overload error. I don't know if I'm doing something wrong or if it's a bug...
I did all the steps twice, following the instructions very carefully (just using the `greet()` example function provided), but I keep getting this problem.

I get the following error, using `Rust 1.63.0 nightly`, `Flutter 3.0.1` and `flutter_rust_bridge 1.34.2`. I'm Running `macOS 12.4` with ` Xcode 13.3` on an M1 Max device.

```
lib/bridge_generated.dart:106:27: Error: The parameter 'ptr' of the method 'CoreWire.store_dart_post_cobject' has type 'Pointer<NativeFunction<Bool Function(Int64, Pointer<Void>)>>', which does not match the corresponding type, 'Pointer<NativeFunction<Uint8 Function(Int64, Pointer<Void>)>>', in the overridden method, 'FlutterRustBridgeWireBase.store_dart_post_cobject'.
 - 'Pointer' is from 'dart:ffi'.
 - 'NativeFunction' is from 'dart:ffi'.
 - 'Bool' is from 'dart:ffi'.
 - 'Int64' is from 'dart:ffi'.
 - 'Void' is from 'dart:ffi'.
 - 'Uint8' is from 'dart:ffi'.
Change to a supertype of 'Pointer<NativeFunction<Uint8 Function(Int64, Pointer<Void>)>>', or, for a covariant parameter, a subtype.
    DartPostCObjectFnType ptr,
                          ^
../flutter_tools/.pub-cache/hosted/pub.dartlang.org/flutter_rust_bridge-1.34.2/lib/src/basic.dart:153:8: Context: This is the overridden method ('store_dart_post_cobject').
  void store_dart_post_cobject(
       ^

Command PhaseScriptExecution failed with a nonzero exit code
** BUILD FAILED **

Building macOS application...                                           
Exception: Build process failed
```

**Comment by fzyzcjy** [comment]
Duplicate of https://github.com/fzyzcjy/flutter_rust_bridge/issues/478

**Comment by w-ensink** [comment]
@fzyzcjy Thank you, I didn't see that one. Solved it by downgrading to `ffigen 5.0.1` like suggested in #478.

**Comment by fzyzcjy** [comment]
You are welcome :)
